backend/package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "spotifyapiproject",
+  "name": "backend",
```

```suggestion
  "name": "spotifyAPI/backend",
```


just a suggestion. would be nice to prefix the `backend` repo with what it belongs to (`spotifyAPI`)

---

Changing the name to the one you suggested gives me "String does not match the pattern of "^(?:@[a-z0-9-*~][a-z0-9-*._~]*/)?[a-z0-9-~][a-z0-9-._~]*$"."

would "spotifyapi-backend" be a good alternative?

---

backend is fine. i'm not too sure how lerna works but we can leave it as is